doc/modules/antivirus.md

```diff
@@ -42,3 +44,9 @@ antivirus {
   }
 }
 ~~~
+
+### SAVAPI specific details ###
+
+The default SAVAPI configuration has a listening unix socket. You must change this to a TCP socket. The option "ListenAddress" in savapi.conf shows some examples. Per default this module expects the socket at 127.0.0.1:4444. You can change this by setting it in the "servers" variable as seen above.
```

Comment:
lua_tcp works OK with UNIX sockets despite its namesake. Are you sure TCP is needed?